### DISCUSSION TITLE
Allow tilting of the osu!mania playfield along the  z-axis

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#15218] [Ideas]
Not sure if lazer has the functionality to implement something "3d" but this would be a feature a decently large subset of the mania community would enjoy having for comfort reasons, if they come from games such as Sound Voltex (linked below as an example of what im referring to):

https://www.youtube.com/watch?v=O3_1ZusubqA&ab_channel=%EB%AF%B8%EB%9D%BC%EC%9D%B4Mirai

**Comment by smoogipoo** [comment]
This requires osu!framework to provide support for perspective transforms.
